### ISSUE TITLE
Export failing with NEO field

### ISSUE THREAD

**Issue by zroberts** [engram-design/FieldManager#79]
**Description**
Exporting a NEO field currently crashing. 

Getting an "Array to String" conversion error on line 146 of the src/services/Export.php file, 

```
'childBlocks' => Json::decodeIfJson((string)$blockType->childBlocks)
```

**Additional info**
- Plugin version: 2.2.3
- Craft version: 3.6.17
- Multi-site: No

Appreciate any assistance!

**Comment by engram-design** [comment]
Fixed in [2.2.4](https://github.com/verbb/field-manager/releases/tag/2.2.4)

**Comment by reilnuud** [comment]
Sad to say this one has returned–not sure if the issue is in this repo, neo's, or both.

**Additional info**

- Plugin version: 2.2.4
- Neo Version: 2.11.4
- Craft version: 3.7.7
- Multi-site: No

**Comment by engram-design** [comment]
@reilnuud Should be fixed for the next release. To get the fix early, change your `verbb/field-manager` requirement in `composer.json` to:

```
"require": {
  "verbb/field-manager": "dev-craft-3 as 2.2.4",
  "...": "..."
}
```

Then run `composer update`.

However, do note that we're also waiting on https://github.com/spicywebau/craft-neo/pull/489 to have the export updated and importing fields fixed.

**Comment by tekstrand** [comment]
@engram-design any chance we can get this released so we don't have to continue using the dev branch?

**Comment by engram-design** [comment]
@tekstrand Yep, would be good to see if https://github.com/verbb/field-manager/issues/82 needs a fix.

**Comment by bzin** [comment]
Having the same issue but with an SuperTable field with Matrix layout. Other SuperTable fields do export correctly.

```
Unknown Property – [yii\base\UnknownPropertyException](http://www.yiiframework.com/doc-2.0/yii-base-unknownpropertyexception.html)
Getting unknown property: verbb\supertable\models\SuperTableBlockType::fields
```

![CleanShot 2022-10-12 at 13 54 55](https://user-images.githubusercontent.com/382735/195336286-af869fbb-8ffa-4be3-ae06-1a98535c2a9d.png)


* Craft edition & version | Craft Pro 4.2.5.2
* Super Table | 3.0.1
* Multi site Yes

**Comment by engram-design** [comment]
Fixed in [3.0.4](https://github.com/verbb/field-manager/releases/tag/3.0.4)
